EnglishTranslation-master/ETC.tsv

```diff
@@ -25527,7 +25527,7 @@ ETC_20161005_025521	Lamp Magic Stone	램프 마법석
 ETC_20161005_025522	Fallen City104	폐도104				
 ETC_20161005_025523	Fallen City105	폐도105				
 ETC_20161005_025524	[Murmillo Master]{nl} Phelixia	[무르밀로 마스터]{nl}         펠릭시아				
-ETC_20161005_025525	 [Lancer Master]{nl} Noer Parecius	 [랜서 마스터]{nl}노어 파레세우스				
+ETC_20161005_025525	 [Lancer Master]{nl}    Noer Parecius	 [랜서 마스터]{nl}노어 파레세우스				
```

Comment:
why the space?

Reply:
NPC name alignment to their role [Tag] .
As an example, please see below:

```
ETC_20150317_009195	[Item Merchant]{nl}         Mirina	[도구 상인]{nl}    미리나	
ETC_20150317_009196	[Equipment Merchant]{nl}               Dunkel	[장비 상인]{nl}     던켈	
ETC_20150317_009197	[Accessory Merchant]{nl}              Ronesa	[악세서리 상인]{nl}      로네사
```
Where the formatting is the same.